### DISCUSSION TITLE
Will OpenVINO come to Forge? Asking from an old system.

### DISCUSSION THREAD

**Discussion by DaHLiA005** [lllyasviel/stable-diffusion-webui-forge#1691] [Q&A]
I've seen OpenVINO done a lot of good optimisation at FastSDCPU branch of A1111 that would positively impact on weak, older systems (i.e. CPU only or GPU with less than 4GB VRAM). What do you guys think about this and would you implement it in the future?

**Comment by DirkHeche** [comment]
I would like to quickly switch between different configurations and tools, reusing models in an efficient way without wasting VRAM for unnecessary duplicates. OpenVINO seems to be perfect for achieving exactly that.

**Comment by lcretan** [comment]
Hi,

In case of A1111, Intel releases official openvino version, however, I cannot find all thing what Intel did a lot of enhancement.

https://github.com/openvinotoolkit/stable-diffusion-webui
